docs/configure-archive-logs.md

```diff
@@ -59,3 +60,14 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Why doesn't Argo Workflows recommend using this feature?
+
+Argo Workflows log storage facilities are quite basic. It is recommended that you use a combination of:
+
+* A fully featured Kubernetes capable logging facility which will provide you with facilities for indexing, searching and managing of log storage.
+* Use [links](links.md) to connect from the Argo Workflows user interface to your logging facility
+    * Use the `scope: workflow` link to get all the logs for a workflow, using the workflow name in the link `${metadata.name}` and the namespace `${metadata.namespace}`
+    * Use `scope: pod-logs` for those from a specific pod of name `${metadata.name}`
```

Comment:
```suggestion
You can use [links](links.md) to connect the Argo UI to your logging facility's UI:
* Link `scope: workflow` to the logs of a Workflow
* Link `scope: pod-logs` to the logs of a specific Pod of a Workflow
* Parametrize the link with `${metadata.name}`, `${metadata.namespace}`, `${metadata.labels}`, and other available metadata
```

- More simple and direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- "Argo UI" is how it is generally referred to. In the same way, we can say that the link is to the logging tool's _UI_ specifically (one UI to another UI)
- capitalize proper nouns: Workflow, Pod
- there's multiple ways of parametrizing the links, `labels` being the one I've used the most, so let's just suggest a few that can be used for both

Reply:
But actually I think the linking part is better explained with examples rather than verbal descriptions (similar to #12582), so I would suggest changing the bullet points above to YAML examples. We do already have examples in the example configmap though, so maybe those are redundant altogether, and we can just link there.

Reply:
I've linked to it instead as the configmap examples cover it mostly, rather than duplicating them here, but otherwise hopefully done what you've suggested.